src/group-node.cpp

```diff
@@ -173,6 +173,11 @@ namespace graphics {
         }
     }
 
+    osg::Node* GroupNode::getOsgNode() const
+    {
+      return (*(list_of_objects_.begin()))->getOsgNode();
```

Comment:
Is there a reason for returning the first element or is it only because you did not know what to return ?